services/user-svc/internal/organization/organization.go

```diff
@@ -349,3 +407,16 @@ func HandleUserCreatedEvent(ctx context.Context, evt *daprcmn.TopicEvent) (retry
 
 	return false, nil
 }
+
+func ChangeMembershipAdminStatus(db *gorm.DB, userID uuid.UUID, organizationID uuid.UUID, isAdmin bool) error {
```

Comment:
[To reply](https://github.com/helpwave/services/issues/158#issuecomment-1584086207)
The configuration of roles (in that case the `IsAdmin` flag) should not be in concern of `AddUserToOrganization()`, non-admin by default, least privileges.

That `AddUserToOrganization()` is untouched also comes handy for the current implementation of invitations.

Reply:
The is admin isn't done in there, but the renaming of the struct is necessary, because the table changed